Commands/ConfigTrait.php

```diff
@@ -46,12 +46,22 @@ protected function renderConfig(OutputInterface $output, array $config)
 
     /**
      * @param InputInterface $input
-     * @return null|string
+     * @param bool $create
+     * @return string
+     * @throws \Exception
      */
-    protected function getConfigPath(InputInterface $input)
+    protected function getConfigPath(InputInterface $input, $create = false)
     {
+        $configOption = $input->getOption('config');
+        if ($configOption && !file_exists($configOption)) {
+            if ($create) {
+                file_put_contents($configOption, json_encode([]));
```

Comment:
Forget to mention.

Yes, this step only create file with array (json encoded). Then `Config Command` does the rest of logic - put default configuration to this file.